lib/ansible/plugins/lookup/list.py

```diff
@@ -13,6 +13,8 @@
     short_description: simply returns what it is given.
     description:
       - this is mostly a noop, to be used as a with_list loop when you dont want the content transformed in any way.
```

Comment:
was it _mostly_ a noop, or _exactly_ a noop?

Reply:
ok, so based solely on the docs, it feels like this filter just took its input and wrapped it in another layer of list, to avoid the single-level of `with_items` flattening?